.github/actions/linux-build/action.yml

```diff
@@ -0,0 +1,73 @@
+name: "Linux Build"
+description: "Build the C++ library on Linux."
+inputs:
+  config: 
+    description: "Configuration to build (Release or Debug, see CMAKE_BUILD_TYPE)"
+    required: true
+    default: "Release"
+  compiler:
+    description: "Compiler to use (clang or gcc)."
+    required: true
+    default: "gcc"
+  version:
+    description: "Version of the compiler to use (min or latest)."
+    required: true
+    default: "latest"
+  coverage:
+    description: "Turn coverage on (ON or OFF)"
+    required: false
+    default: "OFF"
+  sanitize-ub:
+    description: "Turn on UB sanitzer (ON or OFF)"
+    required: false
+    default: "OFF"
+  sanitize-addr:
+    description: "Turn on address sanitzer (ON or OFF)"
+    required: false
+    default: "OFF"
+runs:
+  using: "composite"
+  steps:
+    - name: Install dependencies
+      shell: bash
+      run: |
+        sudo apt-get -qq update
+        sudo apt-get -qq -y install libhdf5-dev lcov
+    - name: Build
+      shell: bash
+      run: |
+        cd cpp
+        if [[ "${{ inputs.compiler }}" == "gcc" ]]; then
+          if [[ "${{ inputs.version }}" == "min" ]]; then
+            export CC=/usr/bin/gcc-7
+            export CXX=/usr/bin/g++-7
+          else        
+            export CC=/usr/bin/gcc-10
+            export CXX=/usr/bin/g++-10
+          fi
+        elif [[ "${{ inputs.compiler }}" == "clang" ]]; then
+          if [[ "${{ inputs.version }}" == "min" ]]; then
+            export CC=/usr/bin/clang-9
+            export CXX=/usr/bin/clang++-9
+          else        
+            export CC=/usr/bin/clang-12
+            export CXX=/usr/bin/clang-12
```

Comment:
clang++-12 ?

Reply:
@dabele export CXX=/usr/bin/clang-12 Is this intended to be clang and not clang++?